cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/VoltageLevelConversion.java

```diff
@@ -50,6 +50,10 @@ public void convert() {
             missing(bv);
             throw new CgmesModelException(String.format("nominalVoltage not found for %s", bv));
         }
+        if (nominalVoltage == 0) {
```

Comment:
I think that in a first approach, we have to drop these voltage levels. It will lead to errors during calculations.